utils/queryApiGateway.ts

```diff
@@ -263,24 +278,23 @@ export default async function queryApiGateway(req: IncomingMessage, uniqueNumber
   }
```

Comment:
After looking through the code, I actually think we need an additional first `if` statement. Something like: 

```typescript
// If apiData is falsey, throw an error
if (!apiData) {
  const nullError = new Error(`API responded with null (queried with ${uniqueNumber}, responded with ${
        apiData}`)
  logger.log(childLogger, 'error', nullError, 'Unexpected API gateway response')
  throw nullError
}
```